### PR TITLE
add: more multi-provider test coverage

### PR DESCRIPTION
**Pull request by ErinHales**
Remembered that I started adding tests a while back and fixed some issues with coverage

## Motivation

Add better test coverage for sdk code.  This mostly covers multi-provider package which is a dependency of the sdk

## PR Checklist

- [x] Added Tests
- [ ] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/multi-provider/jest.config.js [hunk 1]
```diff
@@ -9,5 +9,7 @@ module.exports = {
       isolatedModules: true,
     },
   },
+  // collectCoverageFrom: [
```
Comment (Imti) [inline packages/multi-provider/jest.config.js:12]: Can remove

**Review by kekonen — APPROVED** [review]
lgtm

**Review by Imti — APPROVED** [review]
🧪

**Comment by Imti** [comment]
Need to resolve some conflicts and need @prestwich to review as well

File: packages/multi-provider/tests/index.test.ts [hunk 2]
```diff
@@ -19,7 +19,8 @@ describe('multi-provider', () => {
   const testProvider = new ethers.providers.JsonRpcProvider(
```
Comment (prestwich) [inline packages/multi-provider/tests/index.test.ts:19]: test provider no longer used. throws a linter warning and TS issue

File: packages/multi-provider/tests/index.test.ts [hunk 3]
```diff
@@ -172,32 +192,32 @@ describe('multi-provider', () => {
...
-
-    // TODO: should return provider?
-    // const connectionB = mp.getConnection('b');
-    // expect(connectionB).to.equal(testSigner);
+    expect(connectionA).toBeDefined();
+    expect(connectionA.address).toEqual(signerAddr);
+
+    // gets provider if only provider is registered
+    const connectionB = mp.getConnection('b');
+    console.log('b', connectionB);
```
Comment (prestwich) [inline packages/multi-provider/tests/index.test.ts:211]: console.log

**Comment by ErinHales** [comment]
- [x] Resolved conflicts (was waiting on Odysseas' PR since that touched code coverage as well)
 - [x] Cleaned up unused code/comments
 - [x] Lint
 - [x] Build

Coverage for all files 80-100%
Screen Shot 2022-10-03 at 11 02 45 AM

**Pull request merged**